### PR TITLE
Project Review

### PR DESCRIPTION
**Pull request by LuisSalas94**
Hi @LuisSalas94 
Good job so far.
I've found some issues that you will need to work on:

- You should complete "alt" attribute which isn't mandatory but is incredibly useful for accessibility — screen readers read this description out to their users so they know what the image means.
- There should be only one h1 tag per document.
-  You could use section tag instead of div for better code readability. 

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### CODE REVIEW

**Review by andersonlebon — CHANGES_REQUESTED** [review]
Hi @LuisSalas94 :wave: , 

### To highlight :champagne: 
- Nice job using the GitHub review feature to summarize your review :heavy_check_mark: 
- You have used polite suggestions :heavy_check_mark: 
- You did great using including the screenshot in your summary :heavy_check_mark: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
- [ ] Please Use inline comments to perform the review directly on where the student should make the changes, Kindly have a look at [this link](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) on how to use inline-comments

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] [OPTIONAL] You can kindly include emojis and gifs to make your code review more human-friendly :alien: 

- [ ] [OPTOINAL] Kindly include useful links for resources to help the student have a good documentation  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -1,158 +0,0 @@
...
-		<link rel="stylesheet" href="css/style.css" />
-	</head>
-
-	<body>
-		<!-- header -->
-		<header id="header">
-			<!-- logo -->
-			<div class="logo">
-				<img src="img/logo.png" id="header-img" alt="" />
-			</div>
```
Comment (LuisSalas94) [inline index.html:24]: You should complete "alt" attribute which isn't mandatory but is incredibly useful for accessibility — screen readers read this description out to their users so they know what the image means.

**Review by medaminedev66 — COMMENTED** [review]
Hi, @LuisSalas94    I am **Amine** your reviewer for this PR 😸 

#### You have done really well in implementing the requirements.
- You've used inline comments ✔️ 

## STATUS: CHANGES REQUESTED ♻️

- [ ] Following the project requirements, You should add at least 2 inline comments.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -1,158 +0,0 @@
...
-				name="email"
-				type="email"
-				placeholder="Enter you email here!!!"
-				required
-			/>
-			<input id="submit" type="submit" value="Get Started" />
-		</form>
-
-    <!-- overview -->
-    <div id="Overview">
```
Comment (LuisSalas94) [inline index.html:54]: You could use section tag instead of div for better code readability.

File: index.html [hunk 3]
```diff
@@ -1,158 +0,0 @@
...
-				id="video"
-				width="560"
-				height="315"
-				src="https://www.youtube.com/embed/rcUkTx8WFvA"
-				title="YouTube video player"
-				frameborder="0"
-				allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-				allowfullscreen
-			></iframe>
-		</div>
```
Comment (LuisSalas94) [inline index.html:89]: You could use section tag instead of div for better code readability.

File: index.html [hunk 4]
```diff
@@ -1,158 +0,0 @@
...
-				<h4>$12.99/month after offer period</h4>
-				<h4>2 account</h4>
-				<hr />
-				<ul>
-					<li>Listen to music ad-free</li>
-					<li>Play anywhere - even offline</li>
-					<li>Ad-free music listening, play offline, on-demand playbackk</li>
-				</ul>
-				<button>Get Started</button>
-			</div>
```
Comment (LuisSalas94) [inline index.html:131]: You could use section tag instead of div for better code readability.

File: index.html [hunk 5]
```diff
@@ -1,158 +0,0 @@
...
-				<p>Enjoy nonstop music.</p>
-			</div>
-
-			<!-- card -->
-			<div class="card">
-				<i class="fas fa-play-circle"></i>
-				<h2>Play any song.</h2>
-				<p>Even on mobile.</p>
-			</div>
-		</div>
```
Comment (LuisSalas94) [inline index.html:75]: You could use section tag instead of div for better code readability.

**Review by AleksandarIvezic — APPROVED** [review]
## Hi @LuisSalas94 👋🏼 , 

### STATUS: Approved!!! ✔️ ✔️ ✔️ 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 🥳 


gif



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**